packages/tooling/fast-tooling/src/web-components/html-render-layer-navigation/html-render-layer-navigation.ts

```diff
@@ -96,12 +97,17 @@ export class HTMLRenderLayerNavigation extends HTMLRenderLayer {
         return new OverlayPosition(pos.top, pos.left, pos.width, pos.height);
     }
 
+    private getTitleForDictionaryId(dataDictionaryId): string | null {
+        const dataDictionaryEntry = this.dataDictionary?.[0][dataDictionaryId];
+        return (
+            dataDictionaryEntry.data[dataSetName] ??
+            this.schemaDictionary?.[dataDictionaryEntry.schemaId].title ??
+            null
+        );
```

Comment:
```suggestion
        return (
            this.schemaDictionary?.[this.dataDictionary?.[0][dataDictionaryId].schemaId]
                .title ?? null
        );
```

Reply:
This suggestion breaks the functionality. The function needs to return the dataDictionary dataSetName if it exists otherwise the schema title if it exists otherwise null.